src/main/webapp/homepage_script.js

```diff
@@ -103,4 +90,13 @@ window.onclick = function(event) {
     if (event.target == modal) {
         modal.style.display = "none";
     }
+}
+
+function adjustControlBar(userLogged){
+    console.log(userLogged);
+	if (userLogged === false) {
+        document.getElementById("categories").style.width = "100%";
+    } else {
+        document.getElementById("categories").style.width = "90%";
+    }
```

Comment:
Is it necessary to adjust the width of the control bar? I think the 10% difference won't be so obvious to the users

Reply:
It makes up for the removed 'add task button' so that the categories bar takes up the whole space. I'm a bit obsessive with aligning things haha so definitely not necessary but I do think it makes it look nicer. I plan on changing the implementation of this on a different PR so that there are no onload onclick functions, so this part will go away with that too :)